doc/ctp_install_guide.md

```diff
@@ -57,8 +57,43 @@
               ctp.sh medium medium    #execute medium twice
               ctp.sh webconsole start #start web console to view sql test results
 
-
-## 2. Install CTP as Regression Test in Linux platform
+## 2. Install CTP in Windows platform  
+* ### Install Visual Studio 2017  
+    Visual Studio is required by  CUBRID/bin/make_locale.bat.  
+    When install Visual Studio 2017, choose `Workloads` tab, in `Windows (3)` section, choose `Desktop development with C++`, then click `Install` or `Modify` to start the installation.  
+    After installation, check system variable `VS140COMNTOOLS`   
+  * If `VS140COMNTOOLS` is not added to the system variables automatically, please add it manually.  
+    ```
+    VS140COMNTOOLS=C:\Program Files (x86)\Microsoft VisualStudio\2017\Community\Common7\Tools\    
+    ``` 
+* ### Install cygwin  
+  * Required packages:  `wget`, `zip`, `unzip`, `dos2unix`, `bc`, `expect`.  
+    Do not choose: `gcc` and `MinGW`  
+  * Package versions required  
+    Unsatisfied versions which will lead to many case failures:  
+    ```
+    grep: 3.0-2  
+    gawk: 4.1.4-3  
+    sed: 4.4-1  
+    ```
+    Satisfied versions:  
+    ```
+    gawk: 4.1.3-1  
+    grep: 3.0-1  
+    sed: 4.2.2-3  
+    ```
+    To install the old versions, please refer to this satisfied versions Install old packages of cygwin.  [Satisfied versions comment reference.](http://jira.cubrid.org/browse/CUBRIDQA-422?focusedCommentId=4747548&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4747548)

```

Comment:
Add related information to this guide.
I don't think jira's content is easy to read. We need guide well formated.